packages/calcite-components/src/components/date-picker-month/date-picker-month.tsx

```diff
@@ -435,34 +490,38 @@ export class DatePickerMonth {
    * @param active.dayInWeek
    * @param active.ref
    * @param key
+   * @param active.currentDay
    */
-  private renderDateDay({ active, currentMonth, date, day, dayInWeek, ref }: Day, key: number) {
-    const isFocusedOnStart = this.isFocusedOnStart();
+  private renderDateDay(
+    { active, currentMonth, currentDay, date, day, dayInWeek, ref }: Day,
+    key: number,
+  ): VNode {
+    const isDateInRange = inRange(date, this.min, this.max);
     const isHoverInRange =
       this.isHoverInRange() ||
       (!this.endDate && this.hoverRange && sameDate(this.hoverRange?.end, this.startDate));
 
     return (
-      <div class="day" key={key} role="gridcell">
+      <div class="day-container" key={key} role="gridcell">
         <calcite-date-picker-day
           active={active}
           class={{
-            "hover--inside-range": this.startDate && isHoverInRange,
-            "hover--outside-range": this.startDate && !isHoverInRange,
-            "focused--start": isFocusedOnStart,
-            "focused--end": !isFocusedOnStart,
+            "current-day": currentDay,
```

Comment:
Can we move these, and other inlined classes, to a `CSS` object? This can be done in a follow-up PR.